app/javascript/availability/libraries_locations.json

```diff
@@ -232,12 +232,12 @@
     "IDEALAB-BK": "Berks - IDEA Lab (Instructional Design Lab)",
     "ILL-BK": "Berks - ILL Office",
     "ILL-BR": "Beaver - ILL Office",
-    "ILLEND": "Copy unavailable, request via Interlibrary Loan",
+    "ILLEND": "Copy unavailable",
```

Comment:
@Smullz622 Looks like this needs to be checked out

```
"ILL": "Checked out",
"ILLEND": "Checked out",
```

Reply:
My mistake, I think I've got them all now. Thanks!

Reply:
no worries, thanks for the fix :)